test/functional/feature_fee_estimation.py

```diff
@@ -132,11 +132,13 @@ def make_tx(wallet, utxo, feerate):
 class EstimateFeeTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 3
+        # whitelist peers to speed up tx relay / mempool sync
+        self.noban_tx_relay = True
         # Force fSendTrickle to true (via whitelist.noban)
```

Comment:
nit: This comment is not relevant anymore. Either replace it with the one right before `self.noban_tx_relay = True` or remove it

Reply:
Nice catch! I'll address it.